dashboard/test/ui/features/platform/signing_in.feature

```diff
@@ -16,6 +16,7 @@ Scenario: Student sign in from code.org
   Then I wait to see "#header_user_menu"
   And I wait until element ".display_name" is visible
   And element ".display_name" contains text "Bob"
+  And I delete the cookie named "_loc_notice"
```

Comment:
This test was flaky and driving me nuts. It would occasionally not be able to press the `signin` button because it would race the language selection modal. The reload was to ensure that the cookie was set that would disable that modal from appearing a second time. Yet, if it reloaded _before_ the modal and cookie happened, it would just do it again.

I just set the cookie instead of reloading.